BitmarkRegistry/Helpers/BitmarkFileUtil.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  FileUtil.swift
+//  BitmarkRegistry
+//
+//  Created by Macintosh on 6/28/19.
+//  Copyright © 2019 thuyentruong. All rights reserved.
+//
+
+import Foundation
+
+class BitmarkFileUtil {
+  static func moveFile(from fileURL: URL, toFolder downloadedFolderURL: URL) throws {
+    let filename = fileURL.lastPathComponent
+    let destinationURL = downloadedFolderURL.appendingPathComponent(filename)
+    let data = try Data(contentsOf: fileURL)
+    try data.write(to: destinationURL, options: [.completeFileProtection, .atomic])
```

Comment:
Can we use [this](https://developer.apple.com/documentation/foundation/filemanager/1414750-moveitem) to do moving item? Reading file data to app's memory is not a good idea.